api/STYLE.md

```diff
@@ -34,6 +34,10 @@ In addition, the following conventions should be followed:
   implementation. These indicate that the entity is not implemented in Envoy and the entity
   should be hidden from the Envoy documentation.
 
+* Use a `[#experimental:]` annotation in comments for messages that are considered experimental
```

Comment:
I thought that generally alpha things weren't considered to be in the threat model.  Maybe we should just tag this alpha instead?

@yanavlasov for thoughts

Reply:
This gets to my comment above https://github.com/envoyproxy/envoy/pull/16210#pullrequestreview-649949859. I think the issue is we can only tag as alpha at file level. @snowp points out that sometimes we want to tag at feature level.

Reply:
Right, but can we call them both alpha, rather than alpha and experimental?  Unless there are actual differences, but I think the description above could apply to either so I'd lean towards calling it alpha and shipping :-)

Reply:
Yes, I'm good with more consistent terminology here, that helps reduce some of the confusion.